frontend/src/scenes/billing/billing-utils.ts

```diff
@@ -222,3 +222,32 @@ export const convertLargeNumberToWords = (
         !previousNum && multipleTiers ? ` ${productType}s/mo` : ''
     }`
 }
+
+export const getProration = ({
+    timeRemaining,
+    timeTotal,
+    amountUsd,
+    hasActiveSubscription,
+}: {
+    timeRemaining: number
+    timeTotal: number
```

Comment:
It's not clear to me what should be passed in here. I assume a number of minutes?

Reply:
It's in milliseconds (based on the timestamps we store the data in) - I can make that more explicit.